03-Node-npm-modules/js/movie.js

```diff
@@ -0,0 +1,33 @@
+define(['director'], function(director) {
+	
+		// Practice - Point 3.
+		function movie(mTitle, mDescription)
+		{
+			this.title = mTitle;
+			this.description = mDescription;
+			this.director = null;
+		}		
+			
+		movie.prototype = {
+
+			// Getters & Setters
+			set: function(property, value) {
+				this[property] = value;
+			},
+
+			get: function(property) {
+				return this[property];
+			},
+
+			setDirector: function(directorName) {
+				this.director = new director(directorName);
+			},
+			// End getters & Setters
+
+			mostrarPelicula: function() {
```

Comment:
Try to keep everything in the same language, english would be the best choice :)